cmd/kube-proxy/app/server.go

```diff
@@ -647,34 +646,3 @@ func detectNodeIPs(ctx context.Context, rawNodeIPs []net.IP, bindAddress string)
 	}
 	return primaryFamily, nodeIPs
 }
-
-// getNodeIP returns IPs for the node with the provided name.  If
-// required, it will wait for the node to be created.
-func getNodeIPs(ctx context.Context, client clientset.Interface, name string) []net.IP {
```

Comment:
er... no, I was saying, I think the right fix is to leave kube-proxy as it is, and remove `GetNodeIP` from `pkg/util/node`.

Reply:
oh, I see, you did that as a separate PR.
I don't think we need to move the code in kube-proxy; it's only used at startup time, so it seems fine to have it in `cmd/kube-proxy`

Reply:
OK, thx, i will close this PR and reserve https://github.com/kubernetes/kubernetes/pull/126621

Reply:
OK, thx, i will close this PR and reserve https://github.com/kubernetes/kubernetes/pull/126621